spec.html

```diff
@@ -12189,6 +12206,93 @@
+        </emu-alg>
+      </emu-clause>
+    </emu-clause>
+
+    <!-- es6num="12.5.8" -->
+    <emu-clause id="sec-prefix-decrement-operator">
+      <h1>Prefix Decrement Operator</h1>
+
+      <!-- es6num="12.5.8.1" -->
+      <emu-clause id="sec-prefix-decrement-operator-runtime-semantics-evaluation">
+        <h1>Runtime Semantics: Evaluation</h1>
+        <emu-grammar>UpdateExpression : `--` UnaryExpression</emu-grammar>
+        <emu-alg>
+          1. Let _expr_ be the result of evaluating |UnaryExpression|.
+          1. Let _oldValue_ be ? ToNumber(? GetValue(_expr_)).
+          1. Let _newValue_ be the result of subtracting the value `1` from _oldValue_, using the same rules as for the `-` operator (see <emu-xref href="#sec-applying-the-additive-operators-to-numbers"></emu-xref>).
+          1. Perform ? PutValue(_expr_, _newValue_).
+          1. Return _newValue_.
+        </emu-alg>
+      </emu-clause>
+    </emu-clause>
+  </emu-clause>
+
+  <emu-clause id="sec-exp-operator">
+    <h1>Exponentiation Operator</h1>
+    <h2>Syntax</h2>
+
+    <emu-grammar>
+      ExponentiationExpression[Yield] :
+        UnaryExpression[?Yield]
+        UpdateExpression[?Yield] `**` ExponentiationExpression[?Yield]
+    </emu-grammar>
+
+    <emu-clause id="sec-exp-operator-static-semantics-isfunctiondefinition">
+      <h1>Static Semantics: IsFunctionDefinition</h1>
+      <emu-see-also-para op="IsFunctionDefinition"></emu-see-also-para>
+      <emu-grammar>
+        ExponentiationExpression :
+          UpdateExpression `**` ExponentiationExpression
+      </emu-grammar>
+      <emu-alg>
+        1. Return *false*.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-exp-operator-static-semantics-isvalidsimpleassignmenttarget">
+      <h1>Static Semantics: IsValidSimpleAssignmentTarget</h1>
+      <emu-see-also-para op="IsValidSimpleAssignmentTarget"></emu-see-also-para>
+      <emu-grammar>
+        ExponentiationExpression :
+          UpdateExpression `**` ExponentiationExpression
+      </emu-grammar>
+      <emu-alg>
+        1. Return *false*.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-exp-operator-runtime-semantics-evaluation">
+      <h1>Runtime Semantics: Evaluation</h1>
+      <emu-grammar>
+        ExponentiationExpression : UpdateExpression `**` ExponentiationExpression
+      </emu-grammar>
+      <emu-alg>
+        1. Let _left_ be the result of evaluating _UpdateExpression_.
+        1. Let _leftValue_ be ? GetValue(_left_).
+        1. Let _right_ be the result of evaluating _ExponentiationExpression_.
+        1. Let _rightValue_ be ? GetValue(_right_).
+        1. Let _base_ be ? ToNumber(_leftValue_).
+        1. Let _exponent_ be ? ToNumber(_rightValue_).
+        1. Return the result of <emu-xref href="#sec-applying-the-exp-operator" title>Applying the ** operator</emu-xref> with _base_ and _exponent_ as specified in <emu-xref href="#sec-applying-the-exp-operator"></emu-xref>.
```

Comment:
@anba thanks again for taking a look at this, your attention is always greatly appreciated :)